packages/uif/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @l2beat/uif
 
+## 0.4.0
```

Comment:
Shouldn't it be 0.3.2 ? Last change also added method and changed the MultiIndexer internal behavior